### ISSUE TITLE
Bugs with appearance on the site

### ISSUE THREAD

**Issue by RinatValiullov** [mikolajdobrucki/ikonate#14]
There are some bugs with appearance of  in the _features_ section

Chrome 73.0.3683.103 (64bit)

![www ikonate com_](https://user-images.githubusercontent.com/9755333/56455046-3601fe80-6373-11e9-8957-ac66b8f4f2cd.png)

**Comment by mikolajdobrucki** [comment]
To limit assets usage, the features section uses emoji for illustration purposes. If you can't view them, this probably means that your OS doesn't support emojis or you have them disabled. This is intended behaviour. 

We will probably replace this emojis with inline SVG icons in the future.

**Comment by RinatValiullov** [comment]
> To limit assets usage, the features section uses emoji for illustration purposes. If you can't view them, this probably means that your OS doesn't support emojis or you have them disabled. This is intended behaviour.
> 
> We will probably replace this emojis with inline SVG icons in the future.

Thanks for exhaustive answer, sir.
That's why I posted this issue. I think different users with different OS shouldn't lose representative interface (like me for example).

**Comment by mikolajdobrucki** [comment]
I completely agree. I'm leaving this issue open and will get back to it once emojis are replaced with a different solution. Thank you a lot for your feedback!

**Comment by muhammadridho** [comment]
or if you interest with emoji mac, you can replace with png, wdyt ?
